### PR TITLE
Windows 11, 2k22: Enable persistent TPMs

### LINKED ISSUE #168: Windows 11 and 2k22 use TPMs without persistent storage
**What happened**:

$subject, this can result in unbootable guests when using BitLocker encryption.

**What you expected to happen**:

Persistent TPM storage to be provided by the Windows 11 and 2k22 preferences to avoid guests using BitLocker encryption becoming unbootable.

**How to reproduce it (as minimally and precisely as possible)**:

- Boot a Windows 11/2k22 VM using the associated preferences
- Use BitLocker encryption within the guest
- Reboot

**Additional context**:

Reported downstream against CNV v4.16.0 https://issues.redhat.com/browse/CNV-39710

**Environment**:
- KubeVirt version (use `virtctl version`): N/A
- Kubernetes version (use `kubectl version`): N/A
- VM or VMI specifications: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from /etc/os-release): N/A
- Kernel (e.g. `uname -a`): N/A
- Install tools: N/A
- Others: N/A

### PR DESCRIPTION
**Pull request by lyarwood**
/cc @0xFelix 

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #168 

https://issues.redhat.com/browse/CNV-39710

**Special notes for your reviewer**:

**Release note**:

```release-note
Persistent TPMs are now provided by the Windows 11 and Windows 2k22 preferences, this requires that the `VMPersistentState` feature gate is enabled and configured correctly.
```

### CODE REVIEW

**Comment by lyarwood** [comment]
/close

A few notes after reading the upstream docs....

https://kubevirt.io/user-guide/virtual_machines/persistent_tpm_and_uefi_state/

This requires the `VMPersistentState` FeatureGate and additional `vmStateStorageClass` configurable be defined before it's usable at the moment. Given the current storage class limitations of a `fs` mode with `rwx` support we can't enable this by default in the Windows preferences just yet. Once block mode support is introduced and the need to define a `vmStateStorageClass` configurable is removed we might revisit this but for now I'm going to close this PR.

Users of the Windows 11 and 2k22 preferences can still opt into using this feature by defining `persistent: true` manually within their `VirtualMachine` definitions.

File: scripts/kubevirt.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 set -ex
 
+export KUBEVIRT_DEPLOY_NFS_CSI="${KUBEVIRT_DEPLOY_NFS_CSI:-true}"
```
Comment (0xFelix) [inline scripts/kubevirt.sh:19]: Add a comment here explaining why we need NFS and not Ceph?

**Review by 0xFelix — APPROVED** [review]
Thanks!

/approve
/lgtm

**Pull request merged**